app/models/concerns/international_phone_number_validator.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+# Validates that the international phone number is valid
+class InternationalPhoneNumberValidator < ActiveModel::Validator
+  def validate(record)
+    value = record.international_telephone
+    return if value.blank?
+
+    if value.length > 50
+      record.errors.add(:base, "Value '#{value}' is not a valid international phone number, international phone number can only be up to 50 characters")
+    end
+
+    return if value == value&.gsub(/[^0-9.\-()+ ]/, '')
+
+    record.errors.add(:base, "Value '#{value}' is not a valid international phone number, please only use valid characters which include digits, \
```

Comment:
I think because of the placement of the `\`, this ends up with some extra spaces. So the message logged for me ends up looking like:
```
"Value 'hello' is not a valid international phone number, please only use valid characters which include digits,                              \".\", \"-\", \"(\", \")\", \"+\", \" \".\""
```

Reply:
https://github.com/SaraAlert/SaraAlert/pull/1069/commits/c44c099e7284dd1335eed9023d81f655f7a29685